README.md

```diff
@@ -24,10 +24,50 @@ All contracts on Chainlink must be funded using the local utility token LINK.
 
 ## Network Details
 
-Chainlink oracle address: 0x17dED59fCd940F0a40462D52AAcD11493C6D8073
-Job ID:
+### Ethereum Mainnet (Chain ID: 1)
 
-- indexName: cd7bea47b73a4d47a5cedecf00b9b561
-- indexValue: d06b539cdbeb450aa72f2e70223078af
-- aDayChange: 3d2a6108056644c38f98d755536eb5ac
-- aMonthChange: b871377caf1f4ccb851189f8d906c277
+- Payment amount: 0.01 LINK
+- Oracle address: 0xfE2dD37BC29f5fc4E0cad8F58F4Dbf4AddD5A59A
+- LINK token address: 0x514910771af9ca656af840dff83e8264ecf986ca
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+### Ethereum Rinkeby Testnet (Chain ID: 4)
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0x17dED59fCd940F0a40462D52AAcD11493C6D8073
+- LINK token address: 0x01BE23585060835E02B77ef475b0Cc51aA1e0709
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+### BNB Smart Chain Mainnet (Chain ID: 56)
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0x02a1BE5682f4Fcc941746e95f095c356A7f4D480
+- LINK token address: 0x404460c6a5ede2d891e8297795264fde62adbb75
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+**Important**
+**The LINK provided by the [BNB Chain Bridge](https://www.bnbchain.world/en/bridge) is not ERC-677 compatible, so cannot be used with Chainlink oracles. However, it can be [converted to the official LINK token on BNB Chain using Chainlink's PegSwap service](https://pegswap.chain.link/?_ga=2.171353062.756683581.1650345653-1161789045.1649048909).**
+
+### BNB Smart Chain Testnet (Chain ID: 97)
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0x17dED59fCd940F0a40462D52AAcD11493C6D8073
+- LINK token address: 0x84b9b910527ad5c03a9ca831909e21e236ea7b06
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+### Polygon Mainnet (Chain ID: 137)
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0xA96474C1A08374EFd0F3C9BC7153FDA7A6c8d9e1
+- LINK token address: 0xb0897686c545045afc77cf20ec7a532e3120e0f1
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+**Important**
+**The LINK provided by the [Polygon (Matic) Bridge](https://wallet.polygon.technology/bridge) is not ERC-677 compatible, so cannot be used with Chainlink oracles. However, it can be [converted to the official LINK token on Polygon (Matic) using Chainlink's PegSwap service](https://pegswap.chain.link/?_ga=2.130074441.756683581.1650345653-1161789045.1649048909)**
+
+### Mumbai Testnet (Chain ID: 80001)
```

Comment:
Specify Polygon?

Reply:
Only for polygon mainnet.

Reply:
Renamed to Mumbai (Polygon) Testnet as well